packages/corvid-cli/src/ensureNodeVersion.js

```diff
@@ -0,0 +1,19 @@
+"use strict";
+
+const getMessages = require("./messages");
```

Comment:
we can just decide we're hard coding the message in this file so we don't "leak" the old node version support to other places (instead of getting the message from the ./messages) file